test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        assertIsNotNull();
+        assertIsNotNull("abc");
+        assertIsNotNull("");
+        assertIsNotNull("abc123", "abcdef", "ghi");
+
+        assertIsNull((Object) null);
+        assertIsNull( null, null);
```

Comment:
Do consider cases where there are nulls and non-nulls together. What about lists? Will nulls within a list return true or false?